sumologic-service/0.15.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,31 @@
+version: 0.15.0
+name: sumologic-service
+displayName: sumologic-service
+createdAt: '2022-07-29T18:15:46Z'
+description: Sumo Logic SLI provider for Keptn
+digest: '2022-07-29T23:48:51Z'
+license: Apache-2.0
+homeURL: https://keptn.sh/docs/integrations/
+keywords:
+- keptn
+- sandbox
+- observability
+- monitoring
+links:
+- name: Source
+  url: https://github.com/keptn-sandbox/sumologic-service/tree/release-0.15.0
+annotations:
+  keptn/org: sandbox
+  keptn/kind: observability
+  keptn/version: 0.15.x
```

Comment:
Any versions > 0.15 supported?

Reply:
Not as of now but I plan to support more recent versions.